packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -451,7 +457,7 @@ export abstract class SharedObjectCore<TEvent extends ISharedObjectEvents = ISha
 				this.reSubmit(content, localOpMetadata);
```

Comment:
Should this also decode handles, and then the default implementation re-encodes them?

Reply:
Ping @connorskees, @anthony-murphy

Reply:
And, would be excellent to get test coverage of roundtripping through these scenarios (resubmit and stashed op, maybe something with rollback)

Reply:
this shouldn't matter. the content is the previously submitted content, so it is already parsed. the dds then resubmits anything they want to be submitted which will subsequently get parses in the submit path again